test/gpu/native/streamPrototype/stream.chpl

```diff
@@ -0,0 +1,211 @@
+//
+// Use standard modules for Block distributions, Timing routines, Type
+// utility functions.
+//
+use Time, Types /*, Random */;
+
+//
+// Use shared user module for computing HPCC problem sizes
+//
+use HPCCProblemSize;
+
+//
+// The number of vectors and element type of those vectors
+//
+const numVectors = 3;
+type elemType = real(64);
+
+//
+// Configuration constants to set the problem size (m) and the scalar
+// multiplier, alpha
+//
+
+// TODO : we need the dynamic scope checking, because there is a reduction in
+// `computeProblemSize`
+config const m = 1024, /*computeProblemSize(numVectors, elemType),*/
+             alpha = 3.0;
+
+//
+// Configuration constants to set the number of trials to run and the
+// amount of error to permit in the verification
+//
+config const numTrials = 10,
+             epsilon = 0.0;
+
+//
+// Configuration constants to indicate whether or not to use a
+// pseudo-random seed (based on the clock) or a fixed seed; and to
+// specify the fixed seed explicitly
+//
+/*
+config const useRandomSeed = true,
+             seed = if useRandomSeed then SeedGenerator.oddCurrentTime else 314159265;
+ */
+
+//
+// Configuration constants to control what's printed -- benchmark
+// parameters, input and output arrays, and/or statistics
+//
+config const printParams = true,
+             printArrays = false,
+             printStats = true;
+
+//
+// The program entry point
+//
+proc main() {
+  printConfiguration();   // print the problem size, number of trials, etc.
+
+  on here.getChild(1) {
+    //
+    // ProblemSpace describes the index set for the three vectors.  It
```

Comment:
This comment should be updated to not talk about Block distribution. Or just removed altogether.

Reply:
I adjusted this comment and the one below.